src/components/prescriptions/PrescriptionAddComponent.vue

```diff
@@ -0,0 +1,225 @@
+                >
+            </div>
+        </form>
+    </div>
+</template>
+
+<script setup>
+    import ButtonComponent from '../ButtonComponent.vue';
+    import BouncingBallsComponent from '../BouncingBallsComponent.vue';
+    import ErrorInfo from '../ErrorInfo.vue';
+    import getDoctorsArray from '../../composables/getDoctor.js';
+    import getMedicineArray from '../../composables/getMedicine.js';
+
+    import { computed, ref } from 'vue';
+    import useUserStore from '../../stores/userStore.js';
+    import { timestamp } from '../../firebase/config.js';
+    import useCollection from '../../composables/useCollections.js';
+
+    const { isLoading, addDoc, error } = useCollection('listOfRecipies');
+
+    const nameAndSurnameDoctor = ref(null);
+    const medicine = ref(null);
+    const frequency = ref(null);
+    const researchDate = ref(null);
+    const successFlag = ref(false);
+    const userStore = useUserStore();
+
+    const disableAddTest = computed(() => !dateFormat.value);
+    const dateFormat = computed(
+        () =>
+            researchDate.value &&
+            researchDate.value.split('-').reverse().join('-')
+    );
+
+    const addTest = async () => {
+        try {
+            await addDoc({
+                userId: userStore.getUserId,
+                date: dateFormat.value,
+                doctorData: nameAndSurnameDoctor.value,
+                medicineName: medicine.value,
+                frequencyMedicine: frequency.value,
+                createdAt: timestamp(),
+            });
+            if (error.value) throw new Error();
+            isLoading.value = false;
+            successFlag.value = true;
+        } catch (err) {
+            console.log(err);
+            console.log(error.value);
+            console.log(error.message);
+        } finally {
+            nameAndSurnameDoctor.value = null;
+            medicine.value = null;
+            frequency.value = null;
+            researchDate.value = null;
+        }
+    };
+</script>
+
+<style lang="scss" scoped>
+    .prescriptionBox {
+        @include flex-position(column, nowrap, space-between, flex-start);
+        height: 100%;
+        &__header {
+            @include text-header3($font-weight-semiBold);
+            color: $blue-900;
+        }
+        &__form {
+            height: 100%;
+            select,
+            input {
+                @include text-header6($font-weight-regular);
+            }
+            label {
+                @include label;
+                color: $blue-900;
+            }
+            .actionsAndInfo {
+                @include flex-position(column, nowrap, center, center);
+                gap: 1rem;
+                align-self: center;
+            }
+        }
+        .prescriptionForm {
+            width: 100%;
+            @include flex-position(column, nowrap, space-evenly, flex-start);
+            gap: 1rem;
+            &__bloodSection {
```

Comment:
Same problem. Why you've copied this block of styles without checking class names in html structure? See abouve. In template of your component there is no any html element which has got class: ".prescriptionForm__bloodSection". Ampersand character is to derrieve superrior className of parent element. Thanks to this you can only write & and changing className value. But as you can see there is no any element with bloodSection. Please correct this. See also for learning article: https://css-tricks.com/the-sass-ampersand/

Reply:
Also this selector will not apply beacuse in this component vue does not see element with this class